src/js/utils/CustomConfirmModal.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiCloseFill } from 'react-icons/all';
+
+import Button from 'components/form-elements/Button';
+import Translate from 'utils/Translate';
+
+import 'utils/utils.scss';
+
+const CustomConfirmModal = ({
+  labels: { title, content },
+  onClose,
+  buttons,
+  className,
+}) => (
+  <div className={`d-flex flex-column custom-modal-content justify-content-between bg-white ${className}`}>
+    <div className="d-flex justify-content-between">
+      {(title?.label && title?.default) && (
```

Comment:
nitpicky: reminder - the `&&` should be placed at the beginning of a line:
![Screenshot from 2024-01-12 17-36-05](https://github.com/openboxes/openboxes/assets/93163821/2893784b-46ce-4de9-a508-96a7ab356272).
The ESLint didn't shout there probably due to the `(`.